pkg/sql/randgen/schema.go

```diff
@@ -464,7 +466,7 @@ func GenerateRandInterestingTable(db *gosql.DB, dbName, tableName string) error
 // randColumnTableDef produces a random ColumnTableDef for a non-computed
 // column, with a random type and nullability.
 func randColumnTableDef(rng *rand.Rand, tableIdx int, colSuffix string) *tree.ColumnTableDef {
-	g := randident.NewNameGenerator(&nameGenCfg, rng, fmt.Sprintf("col%d_", tableIdx))
+	g := randident.NewNameGenerator(&nameGenCfg, rng, fmt.Sprintf("col%d", tableIdx))
```

Comment:
GenerateOne down below will add a `_` to the suffix. This would result in column names like `col3__w0_234`. This just fixes that part